README.md

```diff
@@ -1,27 +1,41 @@
 # gw-conditions
-Similar to gage-conditions-gif but for groundwater!
 
-### DISCLAIMER: 
+Show groundwater conditions as little peaks above or below their normal values. 
 
-THE ANALYSIS IN THIS VIZ REPO (AS IT CURRENTLY STANDS 12/23/2020) IS NOT MEANT TO BE FINAL. JUST USED AS AN EXAMPLE OF HOW TO BUILD A VIDEO-BASED DATAVIZ USING SCIPIPER
+## Generate the data behind the visualization and push to S3
+
```

Comment:
Maybe add a little more context here, like:
This animation is built using an R-based pipeline for data processing, and Vue.js + D3.js to create an animated site. The R-based pipeline (1) calculates daily percentiles for each well based on the historic record at each site, and (2) writes an svg map for the base of the animation. This pipeline leverages an internal package, `library(scipiper)` to automate the workflow. You do not need to run the pipeline to be able to build the app locally.